### PR TITLE
Auto translate

### LINKED ISSUE #4: Repeated translation tag
Tag gets repeated if multiple lines, or lines separated by emojis are translated.

### PR DESCRIPTION
**Pull request by quinnha**
Improved `translateComments` with various fixes

## Description
Optimized `translateComments` to better target comment components and eliminate language detect API calls since translating costs the same (and can detect languages!)

Added one translation tag per comment at the bottom of the comment (see issue #4)

Also cleaned up code a little bit for readability (too many brackets lol), and simplified where needed.

### CODE REVIEW

File: scripts/content.js [hunk 1]
```diff
@@ -45,56 +62,63 @@ function callTranslateAPI(text) {
  * @param  {object} mutationsList  text element object.
  * @param  {object} observer  observer properties.
  */
-function translateComments(mutationsList, observer) {
+async function translateComments(mutationsList, observer) {
   for (const mutation of mutationsList) {
     if (mutation.type == "childList") {
-      mutation.addedNodes.forEach((node) => {
+      for (const node of mutation.addedNodes) {
         if (node.nodeName.toLowerCase() == "ytd-comment-renderer") {
```
Comment (wastadtlander) [inline scripts/content.js]: Why is the `toLowerCase` method required here?

File: scripts/content.js [hunk 2]
```diff
@@ -106,20 +130,24 @@ commentWrapper.addEventListener("click", (event) => {
...
     })
-    .catch((error) => {});
+    .catch((error) => {
+      console.log(error);
+    });
 });
 
 /**
- * Observer to translate new comments on load
+ * Observer setup
```
Comment (wastadtlander) [inline scripts/content.js]: One line this.

**Review by wastadtlander — APPROVED** [review]
Looks nice.

File: scripts/content.js (hunk 1 above)
Comment (quinnha) [reply scripts/content.js:69]: good question, will remove

File: scripts/content.js (hunk 2 above)
Comment (quinnha) [reply scripts/content.js]: boooo

File: scripts/content.js (hunk 1 above)
Comment (quinnha) [reply scripts/content.js]: `toLowerCase` helps with readibility, the classes are written in lowercase, so just helps with matching

alternatively, we can just do: `node.nodeName == "YTD-COMMENT-RENDERER"` but that is ugly, let me know which one you want

**Comment by quinnha** [comment]
good to merge, just fixed a small issue with translation tag showing when no translation occurred

also removed detectURL since we arent using it anymore

File: scripts/content.js (hunk 1 above)
Comment (wastadtlander) [reply scripts/content.js]: yeah u right caps is ugly.

**Pull request merged**